python/lsst/phosim/utils/phosim_repackager.py

```diff
@@ -257,60 +256,87 @@ def repackage_eimage(self, phosim_eimg_file, out_dir='.'):
             PhoSim eimage file.
         out_dir : str, optional
             Output directory (the default is '.'.)
+        verbose: bool, optional
+            Set to True to print out processing information.
         """
+        TELCODE = self.telcode
+        CONTRLLR = self.contrllr
 
         sensor = fits.open(phosim_eimg_file)[0]
 
-        # Set keywords in primary HDU, extracting most of the relevant
-        # ones from the original phosim eimage file.
+        filename = phosim_eimg_file
+        raft = os.path.basename(filename).split('_')[4]
+        ccdslot = os.path.basename(filename).split('_')[5]
+        sensorId = '%s_%s'%(raft, ccdslot)
+        print('Repackaging sensor %s '%(sensorId))
+
+        detectors = self.camera.get(sensorId)
+
+        # Set BOT-like keywords in the HDU
+        sensor.header['EXPTIME'] = sensor.header['EXPTIME']
+        sensor.header['DARKTIME'] = sensor.header['DARKTIME']
         sensor.header['RUNNUM'] = sensor.header['OBSID']
-        sensor.header['DATE-OBS'] = \
-            astropy.time.Time(sensor.header['MJD-OBS'], format='mjd').isot
+        sensor.header['MJD-OBS'] = sensor.header['MJD-OBS']
```

Comment:
Do you need this line? It looks like `a=a`?

Reply:
Removed line as obsolete.